test/service/VerifyService.test.ts

```diff
@@ -104,5 +106,12 @@ describe('VerifyService', () => {
             platform: `${os.type()} - ${os.release()} - ${os.platform()}`,
         };
         expect(report).to.be.deep.eq(expected);
+        service.logReport(report);
+        expect(() => service.validateReport(report)).to.throw(
+            `There has been an error. Check the report:
+ - NodeVersion  - Error! - ${currentNodeJsVersion} - At least version ${expectedVersions.node} is required. Currently installed version is ${currentNodeJsVersion}. Check https://nodejs.org/en/download/package-manager/
+ - Docker Version  - Error! - ${currentDockerVersion} - At least version ${expectedVersions.docker} is required. Currently installed version is ${currentDockerVersion}. Check https://docs.docker.com/get-docker/
+ - Docker Compose Version  - Error! - ${currentDockerComposeVersion} - At least version ${expectedVersions.dockerCompose} is required. Currently installed version is ${currentDockerComposeVersion}. Check https://docs.docker.com/compose/install/`,
```

Comment:
you seem to be missing test for new (symbolBootstrap) check that was added

Reply:
I didn't fully like having a bootstrap check on bootstrap. it's like checking their own tool. 

The objective of that check is for `symbol-network verify` to check the installed symbol-bootstrap is the correct one. I have refactored this so the bootstrap check (or any other check) can be added with custom actions (symbol-network will add the custom bootstrap check).